test/lib/utils/exitHelper.ts

```diff
@@ -0,0 +1,348 @@
+import { ExitTxInput } from './types';
+import {
+    Address,
+    ChainId,
+    ComposableStableExitQueryResult,
+    ExitBuildOutput,
+    ExitQueryResult,
+    NATIVE_ASSETS,
+    PoolStateInput,
+    TokenAmount,
+    Slippage,
+    Token,
+    UnbalancedExitInput,
+    ZERO_ADDRESS,
+    SingleAssetExitInput,
+    BALANCER_VAULT,
+    ExitInput,
+    ProportionalExitInput,
+} from '../../../src';
+import { sendTransactionGetBalances, TxResult } from './helper';
+import { expect } from 'vitest';
+import { zeroAddress } from 'viem';
+
+type ExitResult = {
+    exitQueryResult: ExitQueryResult;
+    exitBuildOutput: ExitBuildOutput;
+    txOutput: TxResult;
+};
+
+export const sdkExit = async ({
+    poolExit,
+    exitInput,
+    poolStateInput,
+    slippage,
+    testAddress,
+}: Omit<ExitTxInput, 'client'>): Promise<{
+    exitBuildOutput: ExitBuildOutput;
+    exitQueryResult: ExitQueryResult;
+}> => {
+    const exitQueryResult = await poolExit.query(exitInput, poolStateInput);
+    const exitBuildOutput = poolExit.buildCall({
+        ...exitQueryResult,
+        slippage,
+        sender: testAddress,
+        recipient: testAddress,
+    });
+
+    return {
+        exitBuildOutput,
+        exitQueryResult,
+    };
+};
+
+function getTokens(poolStateInput: PoolStateInput): Address[] {
```

Comment:
we can reuse/split out from joinHelper?

Reply:
Yeah, it's the same function, I made it reusable, thanks!